components/post_view/post_list_virtualized/index.js

```diff
@@ -6,17 +6,23 @@ import {connect} from 'react-redux';
 import {
     getPostIdsInChannel,
     makeGetPostIdsAroundPost,
+    getPost,
 } from 'mattermost-redux/selectors/entities/posts';
 import {makePreparePostIdsForPostList} from 'mattermost-redux/utils/post_list';
+import {memoizeResult} from 'mattermost-redux/utils/helpers';
+
+import {getFirstPostId} from 'utils/post_utils.jsx';
 
 import PostList from './post_list_virtualized.jsx';
 
+const getLatestPostId = memoizeResult((postIds) => getFirstPostId(postIds));
```

Comment:
wow this is really confusing cause of the names, either change the names of the functions or add a comment to explain.

I guess `getFirstPostId` gets the very first item in an array which represents the last posted message?

Reply:
Done changed the function names instead to make better sense